src/NorthwindWebsite/Configuration/ServicesConfiguration.cs

```diff
@@ -46,8 +52,60 @@ public static class ServicesConfiguration
         services.AddScoped<ICategoryRepository, CategoryRepository>();
         services.AddScoped<ISupplierRepository, SupplierRepository>();
 
+        services.AddTransient<IEmailSender, EmailSender>();
```

Comment:
I suppose it can be made with Singleton DI scope lifetime to make single HTTP-client creating